src/BestBooks.js

```diff
@@ -82,50 +129,69 @@ class BestBooks extends React.Component {
+        {/* <UpdateModal  showUpdateModal={this.state.showUpdateModal} handleCloseUpdateModal={this.handleCloseUpdateModal} /> */}
         </Carousel.Caption>
-        <Button onClick={() => this.props.deleteBook(this.props.book.id)}>Delete</Button>
       </Carousel.Item >
     )
     )
 
     return (
       <>
         <h2>My Essential Lifelong Learning &amp; Formation Shelf</h2>
+        <Button onClick={this.showAddModal}>Submit a Book</Button>
         <>
           <Container>
-            <Form onSubmit={this.handleBookSubmit}>
-              <Form.Group controlId="title">
-                <Form.Label>
-                  Title
-                </Form.Label>
-                <Form.Control type="text" />
-              </Form.Group>
-              <Form.Group controlId="description">
-                <Form.Label>
-                  Description
-                </Form.Label>
-                <Form.Control type="text" />
-              </Form.Group>
-              <Form.Group controlId="status">
-                <Form.Label>
-                  Status
-                </Form.Label>
-                <Form.Control type="text" />
-              </Form.Group>
-              <Form.Group controlId="email">
-                <Form.Label>
-                  Email
-                </Form.Label>
-                <Form.Control type="text" />
-              </Form.Group>
-              <Button type="submit">Add a Book</Button>
-            </Form>
-          </Container>
+            <Modal
+
+              show={this.state.showAddModal}
+              onHide={this.handleCloseAddModal}
+              centered
+              size="xl"
+            >
+              <Modal.Header closeButton>
+                <Modal.Title>Input Book Info</Modal.Title>
+              </Modal.Header>
+              <Modal.Body>
+
+                <Form onSubmit={this.handleBookSubmit}>
+                  <Form.Group controlId="title">
+                    <Form.Label>
+                      Title
+                    </Form.Label>
+                    <Form.Control type="text" />
+                  </Form.Group>
+                  <Form.Group controlId="description">
+                    <Form.Label>
+                      Description
+                    </Form.Label>
+                    <Form.Control type="text" />
+                  </Form.Group>
+                  <Form.Group controlId="status">
+                    <Form.Label>
+                      Status
+                    </Form.Label>
+                    <Form.Control type="text" />
+                  </Form.Group>
+                  <Form.Group controlId="email">
+                    <Form.Label>
+                      Email
+                    </Form.Label>
+                    <Form.Control type="text" />
+                  </Form.Group>
+                  <Button type="submit">Submit a Book</Button>
+                </Form>
+              </Modal.Body>
+            </Modal>
+          </Container>          
```

Comment:
- Just a heads-up that all of this will go inside a `BookFormModal component` (this was not for lab 13, but it was part of tasks for lab 12) 👍